custom/ewsghana/tests/test_webusers_sync.py

```diff
@@ -57,6 +57,7 @@ def test_create_webuser(self):
         self.assertEqual(TEST_DOMAIN, domain_name)
         self.assertEqual(UserRole.get_read_only_role_by_domain(TEST_DOMAIN)._id,
                          ewsghana_webuser.get_domain_membership(TEST_DOMAIN).role_id)
+        ewsghana_webuser.delete()
```

Comment:
definitely -1 on the giant hammer. fast tests seem more important than preventing failures from affecting other tests